src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/BailCaseFieldDefinition.java

```diff
@@ -173,7 +187,17 @@
     SUPPORTER_4_PASSPORT(
         "supporter4Passport", new TypeReference<String>(){}),
     FINANCIAL_AMOUNT_SUPPORTER_4_UNDERTAKES(
-        "financialAmountSupporter4Undertakes", new TypeReference<String>(){})
+        "financialAmountSupporter4Undertakes", new TypeReference<String>(){}),
+    GROUNDS_FOR_BAIL_REASONS(
+        "groundsForBailReasons", new TypeReference<String>(){}),
+    GROUNDS_FOR_BAIL_PROVIDE_EVIDENCE_OPTION(
+        "groundsForBailProvideEvidenceOption", new TypeReference<YesOrNo>(){}),
+    UPLOAD_BAIL_EVIDENCE_DOCUMENTS(
+        "uploadTheBailEvidenceDocs", new TypeReference<List<IdValue<DocumentWithDescription>>>(){}),
```

Comment:
I think it should be a list of DocumentWithMetaData if we want to capture the details like Date.

Reply:
TypeReference changed to DocumentWithMetadata and added in those classes+tests